Sprint-1/1-key-exercises/2-initials.js

```diff
@@ -5,7 +5,9 @@ let lastName = "Johnson";
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
 
-let initials = ``;
+// let initials = ``;
```

Comment:
Can you please remove any irrelevant code/scratch work/unnecessary console.logs from the PR? This helps keeping the PR clean and easier to follow